index.html

```diff
@@ -70,12 +70,16 @@ <h3 id="advantages">Advantages of wheels</h3>
                 <h2 id="about-list">What is this list?</h2>
                 <p>This site shows the top 360 most-downloaded packages on <a href="https://pypi.org/">PyPI</a> showing which have been uploaded as wheel archives.</p>
                 <ul>
-                    <li><span class="text-success">Green</span> packages offer wheels,</li>
+                    <li><span class="text-success">Green</span> packages offer wheels compatible with the newest Python version (updated shortly before its final release),</li>
```

Comment:
Suggestion: Let's mention the actual version from NEWEST_PYTHON_VER somewhere, please

Reply:
It's displayed when you hover over the green packages on the list. However I agree, it could be made more visible.

Reply:
I amended this on the line 77 of the diff